### PR TITLE
fix(platform): fix wizard generator related issues

### LINKED ISSUE #6488: bug(platform): form container always unifies the layout for form fields
#### Is this a bug, enhancement, or feature request?
Bug
#### Briefly describe your proposal.
Currently, if we have form fields under form group, form group component will override provided properties for form fields:
hintPlacement, editable, labelLayout, noLabelLayout
#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
0.32
#### If this is a bug, please provide steps for reproducing it.
1. Create form group with form fields inside
2. Set `labelLayout="vertical"` for one form field
3. This form field will still have horizontal label layout.

### PR DESCRIPTION
**Pull request by N1XUS**
## Related Issue.
Closes #6488 #6487 #6245

## Description

fix object rendering on summary page
fix overriding layout of form field
fix hiding navigation buttons for wizard generator
added example with object rendering on summary page
added example of external navigation buttons

##### During Implementation
1. Visual Testing:
- [n/a] visual misalignments/updates
- [n/a] check Light/Dark/HCB/HCW themes
- [n/a] RTL/LTR - proper rendering and labeling
- [n/a] responsiveness(resize)
- [n/a] Content Density (Cozy/Compact/(Condensed))
- [n/a] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [n/a] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [n/a] Mouse vs. Keyboard support
- [n/a] Text Truncation
2. API and functional correctness
- [x] check for console logs (warnings, errors)
- [x] API boundary values
- [x] different combinations of components - free style
- [x] change the API values during testing
3. Documentation and Example validations
- [x] missing API documentation or it is not understandable
- [x] poor examples
- [x] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality
- [x] the commit message(s) follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

##### PR Review
- [x] visual misalignments/updates
- [x] check Light/Dark/HCB/HCW themes
- [x] RTL/LTR - proper rendering and labeling
- [x] responsiveness(resize)
- [x] Content Density (Cozy/Compact/(Condensed))
- [x] States - hover/disabled/focused/active/on click/selected/selected hover/press state
- [x] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
- [x] Mouse vs. Keyboard support
- [x] Text Truncation
2. API and functional correctness
- [x] check for console logs (warnings, errors)
- [x] API boundary values
- [x] different combinations of components - free style
- [x] change the API values during testing
3. Documentation and Example validations
- [x] missing API documentation or it is not understandable
- [x] poor examples
- [x] Stackblitz works for all examples
4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

**Review by platon-rov — CHANGES_REQUESTED** [review]

File: libs/platform/src/lib/wizard-generator/components/wizard-summary-step/wizard-summary-section/wizard-summary-section.component.ts [hunk 1]
```diff
@@ -46,19 +47,18 @@ export class WizardSummarySectionComponent implements OnInit {
...
+        private _formGeneratorService: FormGeneratorService,
+        private _cd: ChangeDetectorRef
     ) { }
 
     /**
      * @hidden
      */
-    ngOnInit(): void {
-
+    async ngOnInit(): Promise<void> {
```
[the remark this answers is not in the document]
Comment (N1XUS) [reply libs/platform/src/lib/wizard-generator/components/wizard-summary-step/wizard-summary-section/wizard-summary-section.component.ts:58]: Yes, it's intended. This component don't have additional hooks, so it should be fine

**Review by platon-rov — APPROVED** [review]
Please fix Travis

**Comment by puru-hk** [comment]
@N1XUS Could you please fix this issue #6474. This issue also related to wizard generator

**Review by InnaAtanasova — APPROVED** [review]

**Comment by N1XUS** [comment]
> Please fix Travis



> @N1XUS Could you please fix this issue #6474. This issue also related to wizard generator

@puru-hk I'm not able to reproduce the issue on this branch by reproducing the steps in the video from the issue itself. Can you confirm that it's reproducible on your side?

**Comment by puru-hk** [comment]
> > Please fix Travis
> 
> > @N1XUS Could you please fix this issue #6474. This issue also related to wizard generator
> 
> @puru-hk I'm not able to reproduce the issue on this branch by reproducing the steps in the video from the issue itself. Can you confirm that it's reproducible on your side?

Yes re-verified it is working fine now.

File: libs/platform/src/lib/wizard-generator/components/wizard-generator/wizard-generator.component.html [hunk 2]
```diff
@@ -1,16 +1,19 @@
-<ng-container *ngIf="_wizardCreated">
+<ng-container *ngIf="wizardCreated">
     <fdp-wizard-body
         [navigationButtonLabels]="navigationButtonLabels"
         [goNextButtonTemplate]="goNextButtonTemplate"
         [finishButtonTemplate]="finishButtonTemplate"
+        [summaryPageTemplate]="summaryPageTemplate"
         [responsivePaddings]="responsivePaddings"
-        [hidden]="!_wizardCreated"
+        [hidden]="!wizardCreated"
```
Comment (dimamarksman) [inline libs/platform/src/lib/wizard-generator/components/wizard-generator/wizard-generator.component.html:8]: what the sense is to have `[hidden]="!wizardCreated"` and parent with `*ngIf="wizardCreated"`?

**Review by dimamarksman — APPROVED** [review]

**Pull request merged**